_automating-configurations/api/create-workflow.md

```diff
@@ -72,12 +72,26 @@ PUT /_plugins/_flow_framework/workflow/<workflow_id>?update_fields=true
 You cannot specify both the `provision` and `update_fields` parameters at the same time.
 {: .note}
 
+If a workflow has been provisioned, you can update and reprovision the full template:
```

Comment:
I expect this section to create a merge conflict when #8171 is merged. This section should go above that new content as it's directly related to updating.  I'll resolve conflicts when the time comes.